{{cookiecutter.project_slug}}/.github/workflows/expo-build.yml

```diff
@@ -31,6 +31,19 @@ on:
         default: false
         type: "boolean"
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+      - name: Set name for PR
+        if: success() && steps.findPr.outputs.number
+        run: echo "PR=pr-${PR}" >> $GITHUB_ENV
+        env:
+          PR: {{ "${{ steps.findPr.outputs.pr }}" }}
+    outputs:
+      PR: {{ "${{ env.PR }}" }}
```

Comment:
If it's a `deployment_status` event, we don't need to do this as the event itself tells us the PR, URL, etc.
But we might still need this for `workflow_dispatch` events.

Reply:
We still use `jwalton/gh-find-current-pr` in a bunch of places we don't need to. I have an example on a project or two somewhere showing how to get the value from the deployment event.